bottlerocket/samples/eks/ecs-migration-test.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: testsys.system/v1
+kind: Test
+metadata:
+  name: ${CLUSTER_NAME}-test-1-initial
+  namespace: testsys
+spec:
+  agent:
+    name: ecs-test-agent
+    image: ${ECS_TEST_AGENT_IMAGE_URI}
+    keepRunning: true
+    configuration:
+      clusterName: ${${CLUSTER_NAME}.clusterName}
```

Comment:
escaping was not required?

Reply:
`envsubst` only substitutes variables that it has a value for, so escaping the outer `${` is not necessary in this case.

Reply:
Switched to an `eval` command instead so the outer `${` is now escaped.